src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "login";
+    private static final String DEFAULT_PASSWORD = "password";
+    private static final int DEFAULT_AGE = 18;
+    private static RegistrationService service;
+    private User testUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+        User registeredUser = new User();
+        registeredUser.setLogin(DEFAULT_LOGIN);
+        registeredUser.setPassword(DEFAULT_PASSWORD);
+        registeredUser.setAge(DEFAULT_AGE);
+        service.register(registeredUser);
```

Comment:
let's move `service.register` call to the method, where we test this
also, we should use `daoStorage.add` instead of `service.register`, because we actually test `register` method
also, we use `Storage.people.clear();` in method annotated with `AfterEach`, so looks like `loginAlreadyAdded_NotOk` test shouldn't pass